components/ui/metric.tsx

```diff
@@ -40,7 +40,7 @@ const MetricInfo = React.forwardRef<HTMLDivElement, MetricInfoProps>(
       <PopoverTrigger className="hover:animate-pulse">
         {trigger || (
           <div className="flex items-center gap-2">
-            <span className="line-clamp-1 text-start">{label}</span>
+            <span className="text-nowrap text-start">{label}</span>
```

Comment:
Keep an eye out for issues here.. `line-clamp-1` was also preventing wrapping, but was frequently truncating early with an ellipsis, instead of expanding to fit the text